magellan-sample/src/main/java/com/wealthfront/magellan/sample/Expedition.kt

```diff
@@ -20,7 +20,7 @@ class Expedition : Journey<ExpeditionBinding>(
 
   override fun onCreate(context: Context) {
     appComponent.inject(this)
-    navigator.goTo(FirstJourney(::goToSecondJourney))
+    navigator.setRoot(FirstJourney(::goToSecondJourney))
```

Comment:
This was causing the onCreate to be triggered twice which we don't want and this fixes it :)